governance/polls/Adjust the System Surplus Buffer - April 5, 2021.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Increase the System Surplus Buffer - April 5, 2021
+summary: Signal your support or opposition for adjusting the Surplus Buffer 
+discussion_link: https://forum.makerdao.com/t/signal-request-adjust-the-surplus-buffer-march-april-2021/6979
+vote_type: Ranked Choice IRV
+categories:
+   - Risk Variable
+options:
+   0: Abstain
+   1: 30 million DAI (no change)
+   2: 60 million DAI (+30M)
+   3: 60 million DAI (+30M) while trying to `flap` 25% of the collected fees 
```

Comment:
attention: unsure if `flap` is going to be rendered properly in the voting portal